hpx/components/containers/partitioned_vector/partitioned_vector.hpp

```diff
@@ -732,6 +732,18 @@ namespace hpx
         }
 
     public:
+
+        std::vector< hpx::id_type > get_partitions_ids()
+        {
+            std::vector< hpx::id_type > ids;
+
+            for(auto const part_data : partitions_)
```

Comment:
Sorry, misunderstanding - what I wanted to ask is why you need the `get_copied_data()` functionality above.